api/spec/v1.yaml

```diff
@@ -1226,9 +1226,26 @@ components:
         success:
           type: boolean
           description: Whether this transaction successfully executed.
+        error:
```

Comment:
This PR will add the same to `RuntimeTransaction` too, right?

Filling it out will be very easy; just need to set the fields [here](https://github.com/oasisprotocol/oasis-indexer/blob/5d6573a83d14960994870774829e6749e7ea727e/api/v1/logic.go#L55-L55), and `cr` already contains the data. (Be mindful of nullable `module`, `message`, and overall `error` though.)

Reply:
> This PR will add the same to RuntimeTransaction too, right?

Yes.

The Consensus `Transactions` change came out of the `RuntimeTransaction` original work because we didn't have it somehow.
https://github.com/oasisprotocol/oasis-indexer/blob/8e3c025c90012e0237448ae40f75d22089879bd2/api/spec/v1.yaml#L2071

https://github.com/oasisprotocol/oasis-indexer/blob/8e3c025c90012e0237448ae40f75d22089879bd2/api/v1/logic.go#L57-L63

Reply:
[`module`](https://github.com/oasisprotocol/oasis-sdk/blob/fb741678585c04fdb413441f2bfba18aafbf98f3/client-sdk/go/types/transaction.go#L489) does appear to be guaranteed for `RuntimeTransactions`, so it is a bit unfortunate indeed that we have to be robust given the union of these two types.

Reply:
Eh in that case I'm ok to make it `required` in openapi also? My feeling is it should be even less likely to go missing than `message`. And if we output an empty string once in a blue moon, so be it.
I'm fine whichever way you decide.

Reply:
I went with leaving it as not `required` to unify the `TxError` across `Consensus` and `Runtime`. I don't see the edge case happening, but will stick to `oasis-core` spec when possible.